app/services/winning_bidder_email_sender.rb

```diff
@@ -4,15 +4,21 @@ def initialize(auction)
   end
 
   def perform
-    AuctionMailer
-      .winning_bidder_notification(bidder: winning_bidder, auction: auction)
-      .deliver_later
+    if auction_has_winner?
+      AuctionMailer
+        .winning_bidder_notification(bidder: winning_bidder, auction: auction)
+        .deliver_later
+    end
   end
 
   private
 
   attr_reader :auction
 
+  def auction_has_winner?
+    !WinningBid.new(auction).find.nil?
```

Comment:
`!WinningBid.new(auction).find.nil?` == `WinningBid.new(auction).find.present?` (latter may be easier to understand...)